tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -355,6 +390,57 @@ def _test_keep_or_replace_suspended_nodes(
     assert_num_instances_in_cluster(cluster_name, region, len(static_nodes))
 
 
+def _test_status_check_replacement(
+    remote_command_executor,
+    scheduler_commands,
+    cluster_name,
+    region,
+    partition,
+    num_static_nodes,
+    num_dynamic_nodes,
+    dynamic_instance_type,
+):
+    """Test nodes failing static check are correctly replaced."""
+    logging.info("Testing that nodes failing static check are correctly replaced")
+    job_id = submit_initial_job(
```

Comment:
do we need this job?

Reply:
Probably not if we just use the exact setup at cluster creation. However, having this here can help us easily re-organize different test cases, and can also be useful when running the test on an existing cluster.

If setup at cluster creation is identical to the nodes requested here(which it is currently) this step should be cheap.